sdk/core/Azure.Core/src/Shared/RequestContentHelper.cs

```diff
@@ -94,5 +94,15 @@ public static RequestContent FromObject(object value)
             content.JsonWriter.WriteObjectValue(value);
             return content;
         }
+        public static RequestContent FromObject(BinaryData value)
```

Comment:
should this be named `FromBytes`?

Reply:
Hello @christothes We have defined a function `FromObject` above this to convert different types of data to RequestContent correctly such string, boolean, number, bytes. 
It is an overload of `FromObject` which is to handle `BinaryData` specifically because in `JsonWriter.WriteObjectValue(value)` it will convert `BinaryData` to base64-encoded which is not expected for request content from BinaryData.

So we still need to use `FromObject` as method Name.